test/jdk/com/sun/jndi/dns/ConfigTests/PortUnreachable.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2002, 2018, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The copyright year had been updated in https://github.com/openjdk/jdk/pull/1116
Thanks.

Reply:
Thanks, I've updated my change with a force-push since there were no reviews yet.